harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -9,16 +10,14 @@
 import warnings
 from abc import abstractmethod
 from inspect import signature
-from typing import Any, Callable, Dict, Iterator, List, Optional, Type, Union, cast
+from typing import Any, Callable, Dict, Iterator, List, Optional, Tuple, Type, Union, cast
 
 import numpy as np
 import torch
-import torch.distributed as dist
```

Comment:
I don't think it's generally safe to use `torch.distributed` after importing just `torch`.  If you do `from torch import distributed` or `import torch.distributed`, then you are guaranteed to have the `.distributed` submodule imported.

This can bite you at unexpected times.  See [our workaround](https://github.com/determined-ai/determined/pull/3321) for when torch did this with a `setuptools` submodule, which stopped being imported automatically.

Reply:
changed it back.